script/gen_requirements_all.py

```diff
@@ -58,6 +58,7 @@
     'numpy',
     'paho-mqtt',
     'pexpect',
+    'phue',
```

Comment:
Since now you mock phue in tests - remove this line.

To make sure this works - uninstall phue from your dev enviroment and check that the tests still pass.